ophyd/signal.py

```diff
@@ -189,6 +193,13 @@ def describe_configuration(self):
         return {}
 
 
+class PySignal(Signal):
```

Comment:
`Signal` itself is a Python signal. `Cpt`/`DevSignal` or something else may be preferable, as it actually indicates what the difference is.